src/components/about/About.js

```diff
@@ -113,31 +121,34 @@ const About = () => {
               </span>
             </h2>
             <p>
-              The goal of the Twitter bot on the Blue Witness project is to
-              adequately scrape Twitter for reports of incidents of police
-              violence. This is a complicated process, and inevitably some data
-              gathered will be difficult to verify or at the very worst simply
-              not relate to police incidents at all. As the Blue Witness project
-              grows, the team is refining the training of this model to
-              accurately reflect individual incidents on Twitter. It reaches out
-              to people posting those incidents and automatically ask some
-              follow-up questions, enhancing the Human Rights First
-              administrative team. By having the Ability to quickly update the
-              incident map as well as the data available for download to
-              researchers, journalists, students, and activists. If you'd like
-              to know more, feel free to reach out to HRF on our contact page.
+              The Twitter scrapper on the Blue Witness project is responsible
```

Comment:
I was going to say to just change "scrapper" to "scraper" and I think we're looking good, but I went ahead and committed that change on my own.